suggester/src/main/java/org/opengrok/suggest/SuggesterProjectData.java

```diff
@@ -66,11 +66,13 @@
  */
 class SuggesterProjectData implements Closeable {
 
+    private static final String TMP_DIR_PROPERTY = "java.io.tmpdir";
```

Comment:
I don't have much background on this.

1. Isn't there some builtin constant?
2. What if this property is not set at all? Is it possible to happen?

Reply:
I think leaving the temporary files in Tomcat temporary directory makes more sense. There is usually too much clutter already in system temp dir.

Reply:
> Isn't there some builtin constant?

I haven't found one. I think this is similar to `line.separator` pre Java 8.

> What if this property is not set at all? Is it possible to happen?

I don't think so. Even `Files` (via `TempFileHelper`) in Java are using this property. So if it would not be set then I suppose all hell would break loose.

> I think leaving the temporary files in Tomcat temporary directory makes more sense. There is usually too much clutter already in system temp dir.

This actually is the Tomcat's temporary dir because Tomcat sets this property to custom value.